src/catalog/category-product-rules.md

```diff
@@ -71,25 +73,25 @@ Match products by rule
 : Determines if the list of products in the category is dynamically generated by a category rule. Options: Yes / No
```

Comment:
what's the purpose of the : at the beginning of these sections?

Reply:
That does the format that you can see on the staging page. Originally designed as a term/definition thing. One of the writers made this from a table I think. We could convert it back to a table.... I have mixed thoughts on this.

Reply:
yeah, i wondered a bit why it wasn't a table.  want me to do that real quick?

Reply:
If you want... up to you.

Reply:
I would love to brain storm a way that we can put this info in without tables because the formatting is so limited and it is kind of a "hack" when we use all of the `<br>`s to list the options.